src/app/zwiazek/kalendarz/_components/Dates/Dates.tsx

```diff
@@ -1,4 +1,11 @@
-export const addMonth = (date: Date, offset = 1) =>
+export const addMonth = (date: Date, offset: number) =>
   new Date(date.setMonth(date.getMonth() + offset));
 
-export const daysOfWeek = ["Pon", "Wt", "Śr", "Czw", "Pt", "Sob", "Ndz"];
```

Comment:
Let's stick with `["Pon", "Wt", "Śr", "Czw", "Pt", "Sob", "Ndz"]` after all. Keep it simple. We are only supporting polish right now anyways.

Reply:
Also we could just call it `weekdays`